src/content/protocols/uniswap-v3.md

```diff
@@ -0,0 +1,131 @@
+| UniversalRouter                    | 0x3fC91A3afd70395Cd496C647d5a6CC9D4B2b7FAD |
+| v3StakerAddress                    | 0xe34139463bA50bD61336E0c446Bd8C0867c6fE65 |
+| Timelock                           | 0x1a9C8182C09F50C8318d769245beA52c32BE35BC |
+| GovernorBravo                      | 0x408ED6354d4973f66138C91495F2f2FCbd8724C3 |
+
+## Permission owners
+
+| Name          | Account                                                                            | Type     |
+| ------------- | ---------------------------------------------------------------------------------- | -------- |
+| ProxyAdmin    | [address](https://etherscan.io/address/0xB753548F6E010e7e680BA186F9Ca1BdAB2E90cf2) | Contract |
+| GovernorBravo | [address](https://etherscan.io/address/0x408ED6354d4973f66138C91495F2f2FCbd8724C3) | Contract |
+| TimeLock      | [address](https://etherscan.io/address/0x1a9C8182C09F50C8318d769245beA52c32BE35BC) | Contract |
+
+## Permissions
+
+| Contract                         | Function            | Impact                                                                                                                                                                                                                                    | Owner                                   |
+| -------------------------------- | ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------- |
+| UniswapV3Factory                 | setOwner            | changes the owner to a new address.                                                                                                                                                                                                       | TimeLocked DAO contract (GovernorBravo) |
+| UniswapV3Factory                 | enableFeeAmount     | enables a fee amount for a given tick spacing.                                                                                                                                                                                            | TimeLocked DAO contract (GovernorBravo) |
+| UniswapV3Factory (UniswapV3Pool) | setFeeProtocol      | set the protocol's % share of the fees.                                                                                                                                                                                                   | TimeLocked DAO contract (GovernorBravo) |
+| UniswapV3Factory (UniswapV3Pool) | collectProtocol     | withdraw accumulated protocol fees to a custom address.                                                                                                                                                                                   | TimeLocked DAO contract (GovernorBravo) |
```

Comment:
instead of

UniswapV3Factory (UniswapV3Pool)

it should say

UniswapV3Pool

Reply:
for both rows